src/language-html/utils/html-elements-attributes.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+const htmlElementAttributes = require("html-element-attributes");
+const mapObject = require("./map-object.js");
+const arrayToMap = require("./array-to-map.js");
+
+const HTML_ELEMENT_ATTRIBUTES = mapObject(htmlElementAttributes, arrayToMap);
+
```

Comment:
Can you try use `html-elements-attributes.evaluate.js` as filename to see if the bundle became smaller?

Reply:
I tried it in my local but bundle size didn't became smaller.
```
 Building packages 
parser-html.js..................................................156 kB    DONE  
```

Reply:
Thanks, it's not safe any way, `mapObject` use `Object.create(null)`, but turns to JSON will be an normal object

`HTML_ELEMENT_ATTRIBUTES.prototype` will be truly.